packages/fiori/src/upload-utils/BodyDragAndDrop.js

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
This is upload collection related. Please put that in the name somehow

Reply:
Renamed it to UploadCollectionBodyDnd